ee-9/feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/transaction/client/main/module.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+~ JBoss, Home of Professional Open Source.
+~
+~ Copyright 2021 Red Hat, Inc., and individual contributors
+~ as indicated by the @author tags.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~     http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.transaction.client">
+
+<exports>
+    <exclude path="org/wildfly/transaction/client/_private" />
+</exports>
+
+<resources>
+    <artifact name="${org.wildfly.transaction:wildfly-transaction-client-jakarta}"/>
+</resources>
+
+<dependencies>
+    <!-- Sorted alphabetically -->
+    <module name="java.naming"/>
+    <module name="jakarta.resource.api"/>
+    <module name="jakarta.transaction.api"/>
```

Comment:
I'm going to post in wildfly-dev in a minute with some thoughts about sticking with java.xxx.api for these. (tl;dr; -- I think we should.)

Reply:
We can discuss on https://lists.jboss.org/archives/list/wildfly-dev@lists.jboss.org/thread/LE6CBNDI3GNXWVX4DOZLU5QNPZQWFJ7K/

Reply:
Hello @bstansberry , do you propose to leave the module names to javax. resource.api and javax.transaction.api for now?

Reply:
Made changes to javax.*.api instead of jakarta.*.api

Reply:
+1. Thanks.